In [339]:
import pandas as pd

In [340]:
rawDataDF = pd.read_csv("MP1.csv")

rawDataDF

Timestamp                            Username  \
0    2024/01/24 1:34:18 PM EST        jessica.herring123@gmail.com   
1    2024/01/24 1:36:10 PM EST                gabbyc2303@gmail.com   
2    2024/01/24 1:38:25 PM EST    nathan.brieske@vikings.berry.edu   
3    2024/01/24 1:56:51 PM EST      chris.retana@vikings.berry.edu   
4    2024/01/24 2:06:10 PM EST          TJ.Tracy@vikings.berry.edu   
5    2024/01/24 2:09:10 PM EST      lucas.kinsey@vikings.berry.edu   
6    2024/01/24 3:29:54 PM EST           brandon.head843@gmail.com   
7    2024/01/24 4:01:27 PM EST    Austin.McMahan@vikings.berry.edu   
8    2024/01/24 4:18:34 PM EST                luke.ommen@gmail.com   
9    2024/01/24 7:39:33 PM EST                 premp7462@gmail.com   
10  2024/01/24 10:57:30 PM EST       vpappalardo@vikings.berry.edu   
11   2024/01/25 9:33:06 AM EST         ethan.perlakowski@gmail.com   
12   2024/01/25 9:33:13 AM EST  BAILEY.HENDERSON@VIKINGS.BERRY.EDU   
13  2024/01/25 10:46:13 AM EST       dylan.bloch@vikings.berry.edu   
14   2024/01/29 4:41:15 AM EST                tchikenspa@gmail.com   

    What is your class year?                         What is your major?  \
0                       2024                            Computer Science   
1                       2025                                     biology   
2                       2025              Sport Leadership and Strategy    
3                       2025                                 accounting    
4                       2024                              Data Analytics   
5                       2025                                  Education    
6                       2025                            Computer Science   
7                       2025                                  Accounting   
8                       2024                                Sports Admin   
9                       2025  Computer Science and Creative Technologies   
10                      2025                                    Finance    
11                      2025                              Data Analytics   
12                      2025                       Creative Technologies   
13                      2025  Computer Science and Creative Technologies   
14                      2024                       Creative Technologies   

    What is your sex?  \
0              Female   
1              Female   
2                Male   
3                Male   
4                Male   
5                Male   
6                Male   
7                Male   
8                Male   
9                Male   
10               Male   
11               Male   
12             Female   
13               Male   
14  Prefer not to say   

   On average, how much time a day do you spend on your phone?  \
0                                           4-5 hours            
1                                            >5 hours            
2                                           2-3 hours            
3                                           3-4 hours            
4                                           3-4 hours            
5                                           1-2 hours            
6                                           3-4 hours            
7                                           3-4 hours            
8                                           2-3 hours            
9                                           3-4 hours            
10                                          3-4 hours            
11                                          4-5 hours            
12                                          4-5 hours            
13                                           >5 hours            
14                                          4-5 hours            

   On average, how many hours a day do you spend studying?   \
0                                           3-4 hours         
1                                           1-2 hours         
2                                   

Who: Finding percentages for each "Who" question:

In [341]:
# Percentage of 2024 Class Year

lenRawDataDF = len(rawDataDF)

seniorDataDF = (rawDataDF["What is your class year?"] == 2024).sum()

seniorDataDF/lenRawDataDF

0.26666666666666666

In [342]:
# Percentage of 2025 Class Year 

juniorDataDF = (rawDataDF["What is your class year?"] == 2025).sum()

juniorDataDF/lenRawDataDF

0.7333333333333333

In [343]:
# Percentage of 2026 Class Year 

sophomoreDataDF = (rawDataDF["What is your class year?"] == 2026).sum()

sophomoreDataDF/lenRawDataDF

0.0

In [344]:
# Percentage of 2027 Class Year 

freshmanDataDF = (rawDataDF["What is your class year?"] == 2027).sum()

freshmanDataDF/lenRawDataDF

0.0

In [345]:
# Percentage of Accounting Majors

accountingDataDF = (rawDataDF["What is your major?"] == "Accounting").sum()

accountingDataDF/lenRawDataDF

0.06666666666666667

In [346]:
# Percentage of Computer Science Majors

csDataDF = ((rawDataDF["What is your major?"] == "Computer Science") | (rawDataDF["What is your major?"] == "Computer Science and Creative Technologies")).sum()

csDataDF/lenRawDataDF

0.26666666666666666

In [347]:
# Percentage of Creative Technologies Majors

crtDataDF = ((rawDataDF["What is your major?"] == "Creative Technologies") | (rawDataDF["What is your major?"] == "Computer Science and Creative Technologies")).sum()

crtDataDF/lenRawDataDF

0.26666666666666666

In [348]:
# Percentage of Data Analytics Majors

daDataDF = (rawDataDF["What is your major?"] == "Data Analytics").sum()

daDataDF/lenRawDataDF

0.13333333333333333

In [426]:
# Percentage of Education Majors

eduDataDF = (rawDataDF["What is your major?"] == "Education ").sum()

eduDataDF/lenRawDataDF

0.06666666666666667

In [427]:
# Percentage of Finance Majors

finDataDF = (rawDataDF["What is your major?"] == "Finance ").sum()

finDataDF/lenRawDataDF

0.06666666666666667

In [429]:
# Percentage of Sport Leadership and Strategy Majors

sportLeadDataDF = (rawDataDF["What is your major?"] == "Sport Leadership and Strategy ").sum()

sportLeadDataDF/lenRawDataDF

0.06666666666666667

In [352]:
# Percentage of Sports Admin Majors

sportAdminDataDF = (rawDataDF["What is your major?"] == "Sports Admin").sum()

sportAdminDataDF/lenRawDataDF

0.06666666666666667

In [434]:
# Accounting in lower case

accDataLowerDF = (rawDataDF["What is your major?"] == "accounting ").sum()

accDataLowerDF/lenRawDataDF

0.06666666666666667

In [432]:
# Percentage of Biology Majors

bioDataDF = (rawDataDF["What is your major?"] == "biology").sum()

bioDataDF/lenRawDataDF

0.06666666666666667

In [354]:
# Percentage of Male Sex

maleDataDF = (rawDataDF["What is your sex?"] == "Male").sum()

maleDataDF/lenRawDataDF

0.7333333333333333

In [355]:
# Percentage of Female Sex 

femaleDataDF = (rawDataDF["What is your sex?"] == "Female").sum()

femaleDataDF/lenRawDataDF

0.2

In [435]:
# Percentage of Prefer not to say 

preferDataDF = (rawDataDF["What is your sex?"] == "Prefer not to say").sum()

preferDataDF/lenRawDataDF

0.06666666666666667

The following below are the percentages for my "Who" questions:

2024 Class Year: 26.7%

2025 Class Year: 73.3%

2026 Class Year: 0%

2027 Class Year: 0%

Accounting Major: 13.4%

Computer Science Major: 26.6%

Creative Technologies Major: 26.6%

Data Analytics Major: 13.3%

Education Major: 6.7%

Finance Major: 6.7%

Sport Leadership Major: 6.7%

Sports Admin Major: 6.7%

Biology Major: 6.7%


Male Sex: 73.3%

Female Sex: 20%

Prefer not to say: 6.7%

What: Finding percentages for each "What" question:

On average, how much time a day do you spend on your phone:

In [436]:
# Phone Percentage of <1 hour
lessOnePhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == "<1 hour").sum()

lessOnePhoneDF/lenRawDataDF

0.0

In [437]:
# Phone Percentage of 1-2 hours
oneTwoPhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == "1-2 hours").sum()

oneTwoPhoneDF/lenRawDataDF

0.06666666666666667

In [438]:
# Phone Percentage of 2-3 hours
twoThreePhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == "2-3 hours").sum()

twoThreePhoneDF/lenRawDataDF

0.13333333333333333

In [439]:
# Phone Percentage of 3-4 hours
threeFourPhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == "3-4 hours").sum()

threeFourPhoneDF/lenRawDataDF

0.4

In [440]:
# Phone Percentage of 4-5 hours
fourFivePhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == "4-5 hours").sum()

fourFivePhoneDF/lenRawDataDF

0.26666666666666666

In [441]:
# Phone Percentage of >5 hours
moreFivePhoneDF = (rawDataDF["On average, how much time a day do you spend on your phone?"] == ">5 hours").sum()

moreFivePhoneDF/lenRawDataDF

0.13333333333333333

On average, how many hours a day do you spend studying:


In [442]:
# Study Percentage of <1 hour
lessOneStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == "<1 hour").sum()

lessOneStudyDF/lenRawDataDF

0.13333333333333333

In [443]:
# Study Percentage of 1-2 hours
oneTwoStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == "1-2 hours").sum()

oneTwoStudyDF/lenRawDataDF

0.3333333333333333

In [444]:
# Study Percentage of 2-3 hours
twoThreeStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == "2-3 hours").sum()

twoThreeStudyDF/lenRawDataDF

0.26666666666666666

In [445]:
# Study Percentage of 3-4 hours
threeFourStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == "3-4 hours").sum()

threeFourStudyDF/lenRawDataDF

0.2

In [446]:
# Study Percentage of 4-5 hours
fourFiveStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == "4-5 hours").sum()

fourFiveStudyDF/lenRawDataDF

0.0

In [447]:
# Study Percentage of >5 hours
moreFiveStudyDF = (rawDataDF["On average, how many hours a day do you spend studying? "] == ">5 hours").sum()

moreFiveStudyDF/lenRawDataDF

0.06666666666666667

The following below are the percentages for my "What" questions:

Phone <1 hour: 0%

Phone 1-2 hours: 6.7%

Phone 2-3 hours: 13.3%

Phone 3-4 hours: 40%

Phone 4-5 hours: 26.7%

Phone >5 hours: 13.3%

Study <1 hour: 13.3%

Study 1-2 hours: 33.3%

Study 2-3 hours: 26.7%

Study 3-4 hours: 20%

Study 4-5 hours: 0%

Study >5 hours: 6.7%

When: Finding percentages for each "When" questions:

When do you use your phone most?

In [448]:
# Percentage of in the morning right after waking up
morningExactPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the morning right after waking up").sum()

morningExactPhoneDF/lenRawDataDF

0.06666666666666667

In [449]:
# Percentage of in the morning after getting ready
morningReadyPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the morning after getting ready").sum()

morningReadyPhoneDF/lenRawDataDF

0.0

In [450]:
# Percentage of during lunch
lunchPhoneDF = (rawDataDF["When do you use your phone most?"] == "During lunch").sum()

lunchPhoneDF/lenRawDataDF

0.0

In [451]:
# Percentage of in the morning/afternoon in class
inClassPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the morning/afternoon in class").sum()

inClassPhoneDF/lenRawDataDF

0.0

In [452]:
# Percentage of in the afternoon after class
afterClassPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the afternoon after class").sum()

afterClassPhoneDF/lenRawDataDF

0.4

In [453]:
# Percentage of in the evening before bed
eveningPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the evening before bed").sum()

eveningPhoneDF/lenRawDataDF

0.13333333333333333

In [454]:
# Percentage of in the evening in bed 
bedPhoneDF = (rawDataDF["When do you use your phone most?"] == "In the evening in bed").sum()

bedPhoneDF/lenRawDataDF

0.4

When do you study the most?

In [455]:
# Percentage of in the morning right after waking up 
morningExactStudyDF = (rawDataDF["When do study the most? "] == "In the morning right after waking up").sum()

morningExactStudyDF/lenRawDataDF

0.0

In [456]:
# Percentage of in the morning after getting ready
morningReadyStudyDF = (rawDataDF["When do study the most? "] == "In the morning after getting ready").sum()

morningReadyStudyDF/lenRawDataDF

0.0

In [457]:
# Percentage of during lunch
lunchStudyDF = (rawDataDF["When do study the most? "] == "During lunch").sum()

lunchStudyDF/lenRawDataDF

0.06666666666666667

In [458]:
# Percentage of in the morning/afternoon in class
inClassStudyDF = (rawDataDF["When do study the most? "] == "In the morning/afternoon in class").sum()

inClassStudyDF/lenRawDataDF

0.06666666666666667

In [459]:
# Percentage of in the afternoon after class
afterClassStudyDF = (rawDataDF["When do study the most? "] == "In the afternoon after class").sum()

afterClassStudyDF/lenRawDataDF

0.6

In [460]:
# Percentage of in the evening before bed
eveningStudyDF = (rawDataDF["When do study the most? "] == "In the evening before bed").sum()

eveningStudyDF/lenRawDataDF

0.2

In [461]:
# Percentage of in the evening in bed 
bedStudyDF = (rawDataDF["When do study the most? "] == "In the evening in bed").sum()

bedStudyDF/lenRawDataDF

0.06666666666666667

The following below are the percentages for my "When" questions:

Phone after wake up: 6.7%
 
Phone morning after get ready: 0%
 
Phone during lunch: 0%

Phone in class: 0%

Phone after class: 40%

Phone before bed: 13.3%

Phone in bed: 40%

Study after wake up: 0%

Study morning after get ready: 0%

Study during lunch: 6.7%

Study in class: 6.7%

Study after class: 60%

Study before bed: 20%

Study in bed: 6.7% 

Where: Finding percentages for each "Where" question:

Where do you use your phone the most?

In [462]:
# In my room
roomPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "In my room").sum()

roomPhoneDF/lenRawDataDF

0.8

In [463]:
# At a place where I can also eat
eatPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "At a place where I can also eat").sum()

eatPhoneDF/lenRawDataDF

0.13333333333333333

In [464]:
# In class
classPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "In class").sum()

classPhoneDF/lenRawDataDF

0.0

In [465]:
# With my friends
friendsPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "With my friends").sum()

friendsPhoneDF/lenRawDataDF

0.0

In [466]:
# In the car
carPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "In the car").sum()

carPhoneDF/lenRawDataDF

0.0

In [467]:
# Living Room
livingRoomPhoneDF = (rawDataDF["Where do you use your phone the most?"] == "Living Room").sum()

livingRoomPhoneDF/lenRawDataDF

0.06666666666666667

Where do you study the most?

In [468]:
# In my room
roomStudyDF = (rawDataDF["Where do you study the most?"] == "In my room").sum()

roomStudyDF/lenRawDataDF

0.5333333333333333

In [469]:
# At a place where I can also eat
eatStudyDF = (rawDataDF["Where do you study the most?"] == "At a place where I can also eat").sum()

eatStudyDF/lenRawDataDF

0.0

In [470]:
# In class
classStudyDF = (rawDataDF["Where do you study the most?"] == "In class").sum()

classStudyDF/lenRawDataDF

0.06666666666666667

In [471]:
# With my friends
friendsStudyDF = (rawDataDF["Where do you study the most?"] == "With my friends").sum()

friendsStudyDF/lenRawDataDF

0.0

In [472]:
#In the car
carStudyDF = (rawDataDF["Where do you study the most?"] == "In the car").sum()

carStudyDF/lenRawDataDF

0.0

In [473]:
# At the library
libraryStudyDF = (rawDataDF["Where do you study the most?"] == "At the library").sum()

libraryStudyDF/lenRawDataDF

0.26666666666666666

In [474]:
# McAllister Hall
hallStudyDF = (rawDataDF["Where do you study the most?"] == "McAllister Hall").sum()

hallStudyDF/lenRawDataDF

0.06666666666666667

In [506]:
# In Hackberry
hackStudyDF = (rawDataDF["Where do you study the most?"] == "In Hackberry").sum()

hackStudyDF/lenRawDataDF

0.06666666666666667

The following below are the percentages for my "Where" questions:

Phone in room: 80%

Phone in dining: 13.3%

Phone in class: 0%

Phone with friends: 0%

Phone in car: 0%

Phone in living room: 6.7%

Study in room: 53.3%

Study in dining: 0%

Study in class: 6.7%

Study with friends: 0%

Study in car: 0%

Study in library: 26.7%

Study in McAllister Hall: 6.7%

Study in Hackberry: 6.7%

Why: Finding percentages for each "Why" questions:

Why do you use your phone for the amount of time you do?


In [476]:
# Social media

phoneMediaDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "Social media").sum()

phoneMediaDF/lenRawDataDF

0.5333333333333333

In [477]:
# School related activities

phoneSchoolDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "School related activities").sum()

phoneSchoolDF/lenRawDataDF

0.0

In [478]:
# Talking to frineds/family

phoneTalkDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "Talking to friends/family").sum()

phoneTalkDF/lenRawDataDF

0.26666666666666666

In [479]:
# To distract myself

phoneDistractDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "To distract myself").sum()

phoneDistractDF/lenRawDataDF

0.06666666666666667

In [480]:
# Games

phoneGameDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "Games").sum()

phoneGameDF/lenRawDataDF

0.06666666666666667

In [481]:
# I don't want to be on my phone very much

phoneNoneDF = (rawDataDF["Why do you use your phone for the amount of time you do?"] == "I don't want to be on my phone very much").sum()

phoneNoneDF/lenRawDataDF

0.06666666666666667

Why do you study for the amount of time you do?


In [482]:
# I feel compelled to 

studyCompelledDF = (rawDataDF["Why do you study for the amount of time you do?"] == "I feel compelled to").sum()

studyCompelledDF/lenRawDataDF

0.06666666666666667

In [483]:
# To succeed in my classes

studySucceedDF = (rawDataDF["Why do you study for the amount of time you do?"] == "To succeed in my classes").sum()

studySucceedDF/lenRawDataDF

0.6666666666666666

In [484]:
# Nothing else to do

studyNothingDF = (rawDataDF["Why do you study for the amount of time you do?"] == "Nothing else to do").sum()

studyNothingDF/lenRawDataDF

0.06666666666666667

In [485]:
# It is required

studyRequiredDF = (rawDataDF["Why do you study for the amount of time you do?"] == "It is required").sum()

studyRequiredDF/lenRawDataDF

0.06666666666666667

In [486]:
# I do not enjoy studying

studyNoEnjoyDF = (rawDataDF["Why do you study for the amount of time you do?"] == "I do not enjoy studying").sum()

studyNoEnjoyDF/lenRawDataDF

0.13333333333333333

Why do you use your phone the most during the time you do?


In [487]:
# It is most convient to check everything in the morning

phoneConvientMorningDF = (rawDataDF["Why do you use your phone the most during the time you do?"] == "It is most convient to check everything in the morning").sum()

phoneConvientMorningDF/lenRawDataDF

0.13333333333333333

In [488]:
# I enjoy using my phone while I eat

phoneEatDF = (rawDataDF["Why do you use your phone the most during the time you do?"] == "I enjoy using my phone while I eat").sum()

phoneEatDF/lenRawDataDF

0.13333333333333333

In [489]:
# It is a good distraction in class

phoneDistractionDF = (rawDataDF["Why do you use your phone the most during the time you do?"] == "It is a good distraction in class").sum()

phoneDistractionDF/lenRawDataDF

0.06666666666666667

In [490]:
# It is most convient to check everything at night 

phoneConvientEveningDF = (rawDataDF["Why do you use your phone the most during the time you do?"] == "It is most convient to check everyting at night").sum()

phoneConvientEveningDF/lenRawDataDF

0.6

In [491]:
# phone time before bed to relax after a day of classes and work 

phoneBeforeBedDF = (rawDataDF["Why do you use your phone the most during the time you do?"] == "phone time before bed to relax after a day of classes and work").sum()

phoneBeforeBedDF/lenRawDataDF

0.06666666666666667

Why do you study the most during the time you do? 


In [492]:
# It is the only time I can

studyOnlyDF = (rawDataDF["Why do you study the most during the time you do? "] == "It is the only time I can").sum()

studyOnlyDF/lenRawDataDF

0.3333333333333333

In [507]:
# I feel the most engaged during this time 

studyEngagedDF = (rawDataDF["Why do you study the most during the time you do? "] == "I feel the most engaged during this time").sum()

studyEngagedDF/lenRawDataDF

0.5333333333333333

In [494]:
# I have nothing else to do during this time 

studyNothingDF = (rawDataDF["Why do you study the most during the time you do? "] == "I have nothing else to do during this time").sum()

studyNothingDF/lenRawDataDF

0.06666666666666667

In [495]:
# When my friends study 
studyEngagedDF = (rawDataDF["Why do you study the most during the time you do? "] == "When my friends study").sum()

studyEngagedDF/lenRawDataDF

0.06666666666666667

Why do you like to use your phone the most in your favorite area?


In [510]:
# I get more things accomplished in this area
phoneAccomplishdDF = (rawDataDF["Why do you like to use your phone the most in your favorite area?"] == "I get more things accomplished in this area").sum()

phoneAccomplishdDF/lenRawDataDF

0.2

In [497]:
# There is nothing else to do 

phoneNothingDF = (rawDataDF["Why do you like to use your phone the most in your favorite area?"] == "There is nothing else to do").sum()

phoneNothingDF/lenRawDataDF

0.4

In [498]:
# There are people around me doing similar things

phonePeopleDF = (rawDataDF["Why do you like to use your phone the most in your favorite area?"] == "There are people around me doing similar things").sum()

phonePeopleDF/lenRawDataDF

0.0

In [499]:
# I do not want to do anything else while I am here

phoneAnythingDF = (rawDataDF["Why do you like to use your phone the most in your favorite area?"] == "I do not want to do anything else while I am here").sum()

phoneAnythingDF/lenRawDataDF

0.3333333333333333

In [500]:
# comfortable in bed 

phoneComfyDF = (rawDataDF["Why do you like to use your phone the most in your favorite area?"] == "comfortable in bed").sum()

phoneComfyDF/lenRawDataDF

0.06666666666666667

Why do you like to study the most in your favorite area?


In [501]:
# I feel less distracted while I am here

studyDistractedDF = (rawDataDF["Why do you like to study the most in your favorite area?"] == "I feel less distracted while I am here").sum()

studyDistractedDF/lenRawDataDF

0.4

In [502]:
# It is the only place I can study

studyOnlyPlaceDF = (rawDataDF["Why do you like to study the most in your favorite area?"] == "It is the only place I can study").sum()

studyOnlyPlaceDF/lenRawDataDF

0.06666666666666667

In [503]:
# There are people around me doing similar things

studySimilarDF = (rawDataDF["Why do you like to study the most in your favorite area?"] == "There are people around me doing similar things").sum()

studySimilarDF/lenRawDataDF

0.3333333333333333

In [504]:
# I do not want to do anything else while I am here

studyAnythingDF = (rawDataDF["Why do you like to study the most in your favorite area?"] == "I do not want to do anything else while I am here").sum()

studyAnythingDF/lenRawDataDF

0.0

In [505]:
# I get a lot more accomplished while I study here 

studyAccomplishedDF = (rawDataDF["Why do you like to study the most in your favorite area?"] == "I get a lot more accomplished while I study here").sum()

studyAccomplishedDF/lenRawDataDF

0.2

The following below are the percentages for my "Why" questions:

Why do you use your phone the amount of time you do? 
    Social Media: 53.3%

    School Related Activities: 26.7%

    Talking to friends/family: 0%

    To distract myself: 6.7%

    Games: 6.7%

    I don't want to be on my phone very much: 6.7%

Why do you study for the amount of time you do? 
    I feel compelled to: 6.7%

    To succeed in my classes: 66.7%

    Nothing else to do: 6.7%

    It is required: 6.7%

    I do not enjoy studying: 13.3%

Why do you use your phone the most during the time you do?
    It is most convient to check everything in the morning: 13.3%

    I enjoy using my phone while I eat: 13.3%

    It is a good distraction in class: 6.7%

    It is most convient to check everything at night: 60%

    Phone time before bed to relax after a day of classes and work: 6.7%

Why do you study the most during the time you do?
    It is the only time I can: 0%

    I feel the most engaged during this time: 53.3%

    I have nothing else to do during this time: 6.7%

    When my friends study: 33.3%

Why do you like to use your phone the most in your favorite area?
    I get more things accomplished in this area: 20%

    There is nothing else to do: 40%

    There are poeple around me doing similar things: 0%

    I do not want to do anything else while I am here: 33.3%

    Comfortable in bed: 6.7%

Why do you like to study the most in your favorite area? 
    I feel less distracted while I am here: 40%

    It is the only place I can study: 6.7%

    There are people around me doing similar things: 33.3%

    I do not want to do anything else while I am here: 0%

    I get a lot more accomplished while I study here: 20%
    


How:
    Briefly talk about the methods you used to give the survey and how much time you were given for this project. Then talk about what tools you used to analyze the data.
        You should also mention possible weaknesses of your study here and how this study could be improved

Discussion/Conclusion: Are there any patterns or trends that you saw in the data? It is ok if you didn't. If you did not, mention that.

I sent out my surveys in two different ways to conduct it. I put it into the hackberry discord, and then I sent it out to a couple different individuals. I had let the survey sit for about a week before I grabbed the responses to get the maximum amount of entries (which ended up being 15). I used a function to grab the percentages of each option within each question, and used the same function to get each individual percentage for options. I think a possible weakness of my survey was that it was far too long. I tried to see if I could find a correlation between high phone usage and low studying time, however, I did not end up seeing any apparent correlation. I believe this was because of the fact that I asked way too many questions so people just started clicking random options. 